engine/runtime-compiler-dump-igv/src/main/java/module-info.java

```diff
@@ -0,0 +1,10 @@
+module org.enso.runtime.compiler.dump.igv {
+  requires jdk.internal.vm.compiler;
```

Comment:
This dependency requires `--add-opens`, right? Maybe you want to leave a comment with the exact invocation syntax here.